NightscoutUploadKit/TimeFormat.swift

```diff
@@ -16,6 +16,11 @@ class TimeFormat: NSObject {
     }
 
```

Comment:
I'm not sure why, but the `DeviceStatus.created_at`, coming from Nightscout, is not in the formatted `formatterISO8601`. This is odd as it appears that is what we are uploading. This unexpected format causes clients, like Caregiver, to fail to fetch the DeviceStatus when using NightscoutUploadKit. I think it may have something to do with some processing happening on Nightscout but I didn't dig in too much.

The change here is to fallback to the format that is coming from Nightscout.